src/image_bios.cpp

```diff
@@ -0,0 +1,196 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (C) 2021 YADRO.
+ */
+
+#include "config.h"
+
+#include "image_bios.hpp"
+
+#include "dbus.hpp"
+#include "fwupderr.hpp"
+#include "subprocess.hpp"
+#include "tracer.hpp"
+
+#include <chrono>
+#include <fstream>
+#include <thread>
+
+static const fs::path aspeedScm = "/sys/bus/platform/drivers/aspeed-smc";
```

Comment:
`aspeedScm` -> `aspeedSmc`
also, I believe the coding style said that acronyms must be same-case, so it's probably even `aspeedSMC`.

Reply:
Done